assignments/lambda-classes.js

```diff
@@ -1 +1,95 @@
 // CODE here for your Lambda Classes
+class Person {
+    constructor(props){
+        this.name = props.name;
+        this.age = props.age;
+        this.location = props.location;
+    }
+    speak() {
+        return(`Hello, my name is ${this.name} and I'm from ${this.location}`);
+    }
+}
+
+class Instructor extends Person {
```

Comment:
Down below you refer to this always as Teacher - So to make your code work, rename Instructor to Teacher